packages/browser/core/src/core/auth/StitchAuthListener.ts

```diff
@@ -17,10 +17,39 @@
 import StitchAuth from "./StitchAuth";
 
 /**
- * An interface to be inherited by classes that need to take action whenever a 
- * particular {@link StitchAppClient} performs an authentication event. An 
- * instance of a {@link StitchAuthListener} must be registered with a 
- * {@link StitchAuth} for this to work correctly.
+ * StitchAuthListener is an interface for taking action whenever 
+ * a particular [[StitchAppClient]]'s authentication state changes.
+ *
+ * Implement [[onAuthEvent]] to handle these events. You can register 
+ * your listener with [[StitchAuth]] using [[StitchAuth.addAuthListener]].
+ *
+ * StitchAuth calls registered listeners when:
+ * - a user logs in
+ * - a user logs out
+ * - a user is linked to another identity
+ * - a listener is registered
```

Comment:
[task] add `* - a user is switched `. This will be true very soon when my multi-user PR goes in